src/users/users.service.spec.ts

```diff
@@ -36,53 +42,234 @@ describe("UsersService Unit Tests", () => {
+            expect(user.firstName).toEqual(consts.USER_ONE_DTO.firstName);
+            expect(user.lastName).toEqual(consts.USER_ONE_DTO.lastName);
+            expect(user.title).toEqual(consts.USER_ONE_DTO.title);
+            expect(user.organization).toEqual(consts.USER_ONE_DTO.organization);
 
-    // Tests the findById function
-    it("should findById", async () => {
-        const user = await usersService.findById(userID);
-        expect(user.email).toEqual(consts.TEST_USER.email);
-    });
+            /* The id of the test constant is undefined, so if 
+                create is successful, the id's should not be equal */
+            let idFlag: boolean;
+            if (user.id != consts.USER_ONE_DTO.id)
+                idFlag = false;
+            expect(idFlag).toBe(false);
+
+            // If create is successful, these two values should not be equal
+            let updateFlag: boolean;
+            if (user.updatedAt.valueOf() != consts.USER_ONE_DTO.updatedAt.valueOf())
+                updateFlag = false;
+            expect(updateFlag).toBe(false);
+        });
+
+        // Tests the create function with dto that has no email field
+        it("should test create function with missing email field", async () => {
+            expect.assertions(1);
+            try {
+                await usersService.create(consts.CREATE_USER_DTO_TEST_OBJ_WITH_MISSING_EMAIL_FIELD);
+            }
+            catch (e) {
+                expect(e.message).toBe("notNull Violation: User.email cannot be null");
+            }
+        });
+
+        // Tests the create function with dto that has invalid email field
+        it("should test create function with invalid email field", async () => {
+            expect.assertions(1);
+            try {
+                await usersService.create(consts.CREATE_USER_DTO_TEST_OBJ_WITH_INVALID_EMAIL_FIELD);
+            }
+            catch (e) {
+                expect(e.message).toBe("Validation error: Validation isEmail on email failed");
+            }
+        });
+
+        // Tests the create function with dto that has no password field
+        it("should test create function with missing password field", async () => {
+            expect.assertions(1);
+            try {
+                await usersService.create(consts.CREATE_USER_DTO_TEST_OBJ_WITH_MISSING_PASSWORD_FIELD);
+            }
+            catch (e) {
+                expect(e.message).toBe("data and salt arguments required");
+            }
+        });
 
-    // Tests the findByEmail function
-    it("should findByEmail", async () => {
-        const user = await usersService.findByEmail(consts.TEST_USER.email)
-        expect(user.email).toEqual(consts.USER_ONE_DTO.email);
+        // Tests the create function with dto that has no passwordConfirmation field
+        it("should test create function with missing password confirmation field", async () => {
+            expect.assertions(1);
+            try {
```

Comment:
The try catch tests could be simplified by using https://jestjs.io/docs/en/using-matchers#exceptions